packages/homestar/src/schemas.js

```diff
@@ -62,7 +62,10 @@ export const Receipt = z.object({
 export const Task = z.object({
   op: z.string(),
   rsc: z.string().url(),
-  nnc: z.string(),
+  nnc: z.union([
+    z.string(),
```

Comment:
this should be empty string, not any string according to homestar schema

Reply:
@hugomrdias thanks 👍🏼 is `z.string().length(0)` the correct way to do that with `zod`? That's what I'm assuming based on the [docs](https://zod.dev/?id=strings)

Reply:
yep